wagtail/images/models.py

```diff
@@ -184,12 +184,51 @@ def open_file(self):
     @contextmanager
     def get_willow_image(self):
         with self.open_file() as image_file:
-            yield WillowImage.open(image_file)
+            yield willow.Image.open(image_file)
+
+
+class WagtailImageFieldFile(models.fields.files.ImageFieldFile):
```

Comment:
nitpick: my preference would be for an import
`from django.db.models.fields.files import ImageFieldFile` and then `WagtailImageFieldFile(ImageFieldFile)` now that we're properly namespaced.

Again, one we can take care of when merging.

Reply:
I got it. I also squashed to eliminate some of the review churn.